src/api/app/models/package.rb

```diff
@@ -349,6 +349,28 @@ def kiwi_file_md5
     nil
   end
 
+  def changes_files
+    result = []
+    dir_hash.elements('entry') do |e|
+      result << e['name'] if e['name'] =~ /.changes$/
+    end
+    result
+  end
```

Comment:
I wonder if this might be something the backend could provide. Unrelated to this PR though. Just thinking loud;-)